src/GoslingMetaComponent.tsx

```diff
@@ -1,72 +1,70 @@
-import React, { useRef, useState, useEffect } from 'react';
-import { GoslingComponent, type GoslingRef, type GoslingSpec } from 'gosling.js';
-import { type Datum } from 'gosling.js/dist/src/core/gosling.schema';
-import MetaTable from './MetaTable';
+import React, {useRef, useState, useEffect} from 'react';
+import {GoslingComponent, type GoslingRef, type GoslingSpec} from 'gosling.js';
+import MetaTable, {MetaTableSpec} from './MetaTable';
 import 'higlass/dist/hglib.css';
 import './index.css';
 
-interface MetaSpec {
-    type: 'table'; // just an example of a user spec
-    // ...
+export type MetaSpec = {
+    width: number;
+    height?: number;
+} & MetaTableSpec
+
+interface AlignmentType {
+    type: 'loose' | 'tight';
```

Comment:
The `loose` alignment was not clear to me before looking at the implementation. My understanding is that when it is `loose`, the meta vis is aligned with the entire Gosling Component. I think both `loose` and `tight` are actually "tight" alignments, but their targets are different (the entire component vs. track). Something to think about in terms of naming.

Reply:
I find you might have something in mind that is not yet implemented, but I think at the moment, `type` is not necessary, i.e., when `trackId` is specified, align with the track; otherwise, align with the entire component.

Reply:
You are right! Thanks

Reply:
I will keep it and rename it to "Connection type" to be more consistent with how it was described in the STAR report. However, this concept might change later if we decide to have a more advanced system to align meta visualizations to gosling visualizations